### DISCUSSION TITLE
How to slow down frame rate?

### DISCUSSION THREAD

**Discussion by homerjam** [cables-gl/cables_docs#468] [Answers & Questions] [answered]
Hello,

I'd like to slow the frame rate to 30fps to help with performance on older machines - is there a convenient way to do this with the API?

Thanks

**Comment by pandrr** [comment]
set mainloop fps to 30

**Reply by homerjam** [reply]
Thanks for the swift reply! Can this be done from the embed API - or only during export, I just have a pre-made patch?

**Comment by pandrr** [comment, marked as answer]
while running you mean ?

you can create and connect a variable for fps and set that via code

**Reply by homerjam** [reply]
Yes exactly. Can this be done after the patch has been compiled?

eg.
```js
window.CABLES.patch.setVariable('fpsLimit', 30)
```

**Reply by pandrr** [reply]
yes
